packages/@coorpacademy-components/src/organism/title-and-input/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Title from '../../atom/title';
+import Autocomplete from '../../atom/autocomplete';
+import SelectMultiple from '../../molecule/select-multiple';
+import SelectOpponents from '../select-opponents';
+import propTypes, {TitleAndInputProps} from './types';
+import style from './style.css';
+
+const buildInput = (childType: string, field: any) => {
```

Comment:
```suggestion
const buildInput = (childType: string, field: Autocomplete || SelectMultiple || SelectOpponents) => {
```
c'est pas un any c'est l'un des trois types